homeassistant/components/sensor/melissa.py

```diff
@@ -0,0 +1,98 @@
+"""
+Support for Melissa climate Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.melissa/
+"""
+import logging
+
+from homeassistant.components.melissa import DOMAIN, DATA_MELISSA
+from homeassistant.const import TEMP_CELSIUS, STATE_UNKNOWN
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = [DOMAIN]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the melissa sensor platform."""
+    sensors = []
+    api = hass.data[DATA_MELISSA]
+    devices = api.fetch_devices().values()
+
+    for device in devices:
+        sensors.append(MelissaTemperatureSensor(device, api))
+        sensors.append(MelissaHumiditySensor(device, api))
+    add_devices(sensors)
+
+
+class MelissaSensor(Entity):
+    """Representation of a Melissa Sensor."""
+
+    _type = 'generic'
+
+    def __init__(self, device, api):
+        """Initialize the sensor."""
+        self._api = api
+        self._state = STATE_UNKNOWN
```

Comment:
Init state as `None`. `STATE_UNKNOWN` shouldn't be used in components or platforms. The base entity class will handle unknown state.